src/main/java/javax/security/enterprise/identitystore/IdentityStore.java

```diff
@@ -67,7 +67,7 @@
  */
 public interface IdentityStore {
     
-    public static final Set<ValidationType> DEFAULT_VALIDATION_TYPES = EnumSet.of(VALIDATE, PROVIDE_GROUPS);
+    Set<ValidationType> DEFAULT_VALIDATION_TYPES = EnumSet.of(VALIDATE, PROVIDE_GROUPS);
```

Comment:
Without looking up the language spec I'm not sure about the syntax details, but the compiler seems fine with it, as far as the intent, the idea is that this Set is immutable. The public static final syntax (at least the final part) seemingly achieves that. What's the motivation to remove it?